networking/k8s_nmstate/k8s-nmstate-configuring.adoc

```diff
@@ -0,0 +1,28 @@
+[id="k8s-nmstate-configuring"]
+
+= Configuring
+include::modules/common-attributes.adoc[]
+:context: k8s-nmstate-configuring
+
+The `kubernetes-nmstate` operator enables users to configure network interfaces, DNS settings and routing on {product-title} cluster nodes.
+
+Two object types drive the configuration:
+
+- `NodeNetworkConfigurationPolicy`: This object defines the policy, which is the desired network configuration on cluster nodes.
```

Comment:
We aren't using _desired_.

>desired adjective
Do not use. Use "required" or "the that you want".

Maybe for k8s _intended_ or _target configuration_ is better?